l2-frontend/src/fields/LaboratoryPreviousResults.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div>
+    <table class="table table-bordered table-condensed">
+      <colgroup>
+        <col width='500'/>
+        <col width='500'/>
+        <col width='300'/>
+        <col width='300'/>
+        <col width='300'/>
+        <col width='200'/>
+        <col />
+      </colgroup>
+      <thead>
+      <tr>
+        <th>Анализ</th>
+        <th>Тест</th>
+        <th>Значение</th>
+        <th>Ед. изм</th>
+        <th>Дата</th>
+        <th>Исполнитель</th>
+        <th class="cl-td">
+          <button class="btn btn-blue-nb" @click="delete_rows" :disabled="disabled" v-tippy="{ placement : 'bottom'}"
+                  title="Очистить строки" >
+            <i class="fa fa-times"/>
+          </button>
+        </th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr v-for="(val, index) in tb_data">
+        <td class="cl-td"><input type="text" class="form-control" :disabled=disabled placeholder="Анализ-наименование" v-model="val.researchTitle"></td>
+        <td class="cl-td"><input type="text" class="form-control" :disabled=disabled placeholder="Тест-наименование" v-model="val.fractionTitle"></td>
+        <td class="cl-td"><input type="text" class="form-control" :disabled=disabled placeholder="Значение" v-model="val.value"></td>
+        <td class="cl-td"><input type="text" class="form-control" :disabled=disabled placeholder="Ед. изм" v-model="val.units"></td>
+        <td class="cl-td"><input type="text" class="form-control" :disabled=disabled placeholder="Дата" v-model="val.date"></td>
+        <td class="cl-td"><input type="text" class="form-control" :disabled=disabled placeholder="Исполнитель" v-model="val.docConfirm"></td>
```

Comment:
Почему без кавычек и почему не readonly?
```suggestion
        <td class="cl-td"><input type="text" class="form-control" :readonly="disabled" placeholder="Исполнитель" v-model="val.docConfirm"></td>
```